pkg/tsdb/stackdriver/stackdriver.go

```diff
@@ -465,14 +489,17 @@ func (e *StackdriverExecutor) parseResponse(queryRes *tsdb.QueryResult, data Sta
 		}
 	}
 
-	queryRes.Meta.Set("resourceLabels", resourceLabels)
-	queryRes.Meta.Set("metricLabels", metricLabels)
+	queryRes.Meta.Set("labels", labels)
 	queryRes.Meta.Set("groupBys", query.GroupBys)
-	queryRes.Meta.Set("resourceTypes", resourceTypes)
 
 	return nil
 }
 
+func toSnakeCase(str string) string {
+	matchAllCap := regexp.MustCompile("(.)([A-Z][a-z]*)")
```

Comment:
This could be global, to avoid instantiating the regex object every time.

Reply:
👍